2025-10-18 20:24:27,778 [INFO] app: Helpdesk Pro started
2025-10-18 20:24:29,834 [INFO] app: Helpdesk Pro started
2025-10-18 20:30:38,637 [INFO] app: Helpdesk Pro started
2025-10-18 20:30:39,782 [INFO] app: Helpdesk Pro started
2025-10-18 20:42:07,602 [INFO] app: Helpdesk Pro started
2025-10-18 20:42:36,613 [INFO] app: Helpdesk Pro started
2025-10-18 20:42:51,780 [INFO] app: Helpdesk Pro started
2025-10-18 20:42:57,325 [INFO] app: Helpdesk Pro started
2025-10-18 20:43:43,044 [INFO] app: Helpdesk Pro started
2025-10-18 20:43:44,280 [INFO] app: Helpdesk Pro started
2025-10-18 20:47:06,866 [INFO] app: Helpdesk Pro started
2025-10-18 20:47:08,265 [INFO] app: Helpdesk Pro started
2025-10-18 20:51:07,922 [INFO] app: Helpdesk Pro started
2025-10-18 20:51:09,157 [INFO] app: Helpdesk Pro started
2025-10-18 20:55:52,314 [INFO] app: Helpdesk Pro started
2025-10-18 20:55:55,914 [INFO] app: Helpdesk Pro started
2025-10-18 20:57:42,880 [INFO] app: Helpdesk Pro started
2025-10-18 20:57:44,086 [INFO] app: Helpdesk Pro started
2025-10-18 21:08:58,312 [INFO] app: Helpdesk Pro started
2025-10-18 21:45:04,351 [INFO] app: Helpdesk Pro started
2025-10-18 21:45:06,727 [INFO] app: Helpdesk Pro started
2025-10-18 21:49:01,468 [INFO] app: Helpdesk Pro started
2025-10-18 21:49:02,665 [INFO] app: Helpdesk Pro started
2025-10-18 21:52:45,719 [INFO] app: Helpdesk Pro started
2025-10-18 21:52:46,885 [INFO] app: Helpdesk Pro started
2025-10-18 21:54:05,072 [INFO] app: Helpdesk Pro started
2025-10-18 21:54:06,163 [INFO] app: Helpdesk Pro started
2025-10-18 22:03:51,282 [INFO] app: Helpdesk Pro started
2025-10-18 22:03:52,423 [INFO] app: Helpdesk Pro started
2025-10-18 22:05:08,574 [INFO] app: Helpdesk Pro started
2025-10-18 22:05:09,661 [INFO] app: Helpdesk Pro started
2025-10-18 22:07:50,349 [INFO] app: Helpdesk Pro started
2025-10-18 22:15:29,884 [INFO] app: Helpdesk Pro started
2025-10-18 22:15:31,029 [INFO] app: Helpdesk Pro started
2025-10-18 22:16:09,156 [INFO] app: Helpdesk Pro started
2025-10-18 22:16:10,255 [INFO] app: Helpdesk Pro started
2025-10-18 22:17:31,761 [INFO] app: Helpdesk Pro started
2025-10-18 22:17:33,207 [INFO] app: Helpdesk Pro started
2025-10-18 22:18:43,042 [INFO] app: Helpdesk Pro started
2025-10-18 22:19:42,373 [INFO] app: Helpdesk Pro started
2025-10-18 22:19:43,544 [INFO] app: Helpdesk Pro started
2025-10-18 22:22:22,293 [ERROR] app: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\app\__init__.py", line 115, in index
    return render_template("index.html")
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\templates\index.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\templates\base.html", line 6, in top-level template code
    <title>{{ _('Helpdesk Pro') }}</title>
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\utils.py", line 92, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
jinja2.exceptions.UndefinedError: '_' is undefined
2025-10-18 22:23:42,548 [ERROR] app: Exception on / [GET]
Traceback (most recent call last):
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\app\__init__.py", line 115, in index
    return render_template("index.html")
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\environment.py", line 1295, in render
    self.environment.handle_exception()
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\templates\index.html", line 1, in top-level template code
    {% extends 'base.html' %}
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\templates\base.html", line 6, in top-level template code
    <title>{{ _('Helpdesk Pro') }}</title>
  File "C:\Users\ioannisb\Documents\Python-Development\IT\helpdesk_pro\.venv\lib\site-packages\jinja2\utils.py", line 92, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
jinja2.exceptions.UndefinedError: '_' is undefined
2025-10-18 22:27:35,244 [INFO] app: Helpdesk Pro started
2025-10-18 22:27:36,707 [INFO] app: Helpdesk Pro started
